README.md

```diff
@@ -1,582 +1,629 @@
-[![Build Status](https://travis-ci.org/spark-jobserver/spark-jobserver.svg?branch=master)](https://travis-ci.org/spark-jobserver/spark-jobserver)
```

Comment:
sorry about the diff, I tried both the Windows and Unix version of the newlines, but it seems that both are different from Master....